src/main/java/uk/gov/ons/ssdc/supporttool/endpoint/JobEndpoint.java

```diff
@@ -168,6 +178,48 @@ public String getErrorDetailCsv(
     return csvContent;
   }
 
+  @PostMapping
+  public ResponseEntity<?> submitUploadJob(
+      @RequestParam(value = "fileId") String fileId,
```

Comment:
```suggestion
      @RequestParam(value = "fileId") UUID fileId,
```

This should make it safer against injection attacks